suclass/cms/templates/head-extra.html

```diff
@@ -0,0 +1,3 @@
+<%namespace name='static' file='/static_content.html'/>
+<link rel="stylesheet" href="${static.url('css/vendor/su-identity.css')}" />
+<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400i,600,700|Source+Sans+Pro:400,400i,600" rel="stylesheet">
```

Comment:
Ah, seems like we're not using the `lagunita` version because it includes GA code?

Seems like we should move that GA code out into an optional include and then this file can be symlinked on `suclass` and the GA file _not_ be present there (only `lagunita`).

Reply:
See: https://github.com/Stanford-Online/openedx-themes/pull/96

Reply:
@caesar2164 You can now rebase over master to pull in my changes from #96 .

Then we can symlink `head-extra.html` but _not_ `head-ga.html`.